### PR TITLE
Rhmap 9643

### PR DESCRIPTION
**Pull request by edewit**
cleaned up double dependencies in dist fixing issue [RHMAP-9643](https://issues.jboss.org/browse/RHMAP-9643)

@nialldonnellyfh review?

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="format-detection" content="telephone=no">
 
-    <script src="browserify.js" type="text/javascript" charset="utf-8"></script>
+    <script src="lib/browserify.js" type="text/javascript" charset="utf-8" concat='true'></script>
```
Comment (jcesarmobile) [inline www/index.html:17]: What is the concat used for? I've tried to look for information about it and didn't find anything.

Nevermind, I found it's used on the Gruntfile.js

**Comment by jcesarmobile** [comment]
Looks good, but as there is a subtask "identify if lib.min.js could be removed" (https://issues.jboss.org/browse/RHMAP-9662). Might be possible to put the lib.min.js code inside the main.js?

**Comment by nialldonnellyfh** [comment]
Has browserify.js been removed?

Seems to be missing from http://testing.ndonnelly2.skunkhenry.com/box/srv/1.1/wid/testing/studio/hnb4gnkpwqsgot5zxbruqisz/file/www/index.html?fh_destination_code=studio&fh_apptitle=RHMAP-9643&url=https://no_cloud_app.example.com

**Comment by jcesarmobile** [comment]
Yes, the browserify.js code should be inside main.js

**Comment by edewit** [comment]
@jcesarmobile this split lib.js main.js was already there, I think it's because some scripts are not uglyfied?

**Comment by jcesarmobile** [comment]
yes lib.min.js has the content of the linked files on lib folder and uglyfied. (it just have mobiscroll and async)
main.js is the content of the linked files on other folders different than lib, and mobiscroll from lib folder (we have it duplicated)

We should fix:
- the mobiscroll duplicate
- browserify.js is not being included in main.js, but it's included in lib.js if you run `grunt browserify` and then `grunt`
- browserify.js has async included, so we duplicate it in lib.js if we do what I said on the previous step.

As async is in browserify.js and mobiscroll is in main.js I don't think we need lib.js, I think we just need to make main.js to include browserify.js.

**Comment by jcesarmobile** [comment]
LGTM

**Pull request merged**